README.md

```diff
@@ -136,6 +136,76 @@ custom:
   datadog:
     enabled: ${self:custom.staged.dd_enabled, true}
 ```
+### Serverless Monitors 
+
+There are seven recommended monitors with default values pre-configured. 
+
+| Monitor              | Metrics                                                                                   | Default Threshold | Serverless Monitor ID |
+|:--------------------:|:-----------------------------------------------------------------------------------------:|:-----------------:|:---------------------:|
+| High Error Rate      | `aws.lambda.errors`/`aws.lambda.invocations`                                              | >= 10%            | `high_error_rate`     |
+| Timeout              | `aws.lambda.duration.max`/`aws.lambda.timeout`                                            | >= 1              | `timeout`             |
+| Out of Memory        | `aws.lambda.lambda.enhanced.max_memory_used`/<br>`aws.lambda.memorysize`                  | >= 1              | `out_of_memory`       |
+| High Iterator Age    | `aws.lambda.iterator_age.maximum`                                                         | >= 24 hrs         | `high_iterator_age`   |
+| High Cold Start Rate | `aws.lambda.enhanced.invaocations(cold_start:true)`/<br>`aws.lambda.enhanced.invocations` | >= 20%            | `high_cold_start_rate`|
+| High Throttles       | `aws.lambda.throttles`/`aws.lambda.invocations`                                           | >= 20%            | `high_throttles`      |
+| Increased Cost       | `aws.lambda.enhanced.estimated_cost`                                                      | &#8593;20%        | `increased_cost`      |
+ 
+#### To Enable and Configure a Recommended Serverless Monitor 
+
+To create a serverless monitor, you must use its respective serverless monitor ID in addition to passing in the API key and Application key in the `serverless.yml` file. If you’d like to further configure the specific parameter values for a recommended monitor, you can directly define parameters below the serverless monitor ID. Parameters not specified under a recommended monitor will use the default recommended value. At this time, the query parameters for recommended monitors cannot be overriden or customized. As such, the critical threshold value and query for recommended monitors will default to the recommended monitor. For further documentation on monitor parameters, see the [Datadog Monitors API](https://docs.datadoghq.com/api/latest/monitors/#create-a-monitor). You can directly update a monitor by changing the parameter values. To delete a monitor, remove the monitor from the `serverless.yml` template. 
```

Comment:
I would clarify that you need to pass in the separate `monitorsApiKey` and `monitorsAppKey` parameters -- it's not enough to just set the regular apiKey parameter as currently written.

Reply:
Also, you mention that the query parameter cannot be overriden or customized. What happens if the customer tries to override the query parameter by accident -- does this fail silently, crash, or something else?

Reply:
It will get overridden by the pre-defined query param. I've updated the documentation to reflect this. Thanks for pointing it out!